Functions/xASL_io_SplitASL_M0.m

```diff
@@ -1,32 +1,61 @@
-function xASL_io_SplitASL_M0(InPath,iM0)
-%xASL_io_SplitASL_M0 Splits ASL & M0 images, when they are within the same NIfTI
+function xASL_io_SplitASL_M0(inPath, iM0, iDummy)
```

Comment:
Would rename this function to `xASL_io_SplitASL`

Reply:
Agree. Done.